<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomerDialog</class>
 <widget class="QDialog" name="CustomerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Customer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="2" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="nameLineEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="surnameLabel">
       <property name="text">
        <string>Surname:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="surnameLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="titleLabel">
       <property name="text">
        <string>Title:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="titleLineEdit"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="streetLineEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="streetLabel">
       <property name="text">
        <string>Street:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="cityLabel">
       <property name="text">
        <string>City:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="cityLineEdit"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="mailLabel">
       <property name="text">
        <string>Mail</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="mailLineEdit"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="organisationLabel">
       <property name="text">
        <string>Organisation:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="organisationLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>organisationLineEdit</tabstop>
  <tabstop>titleLineEdit</tabstop>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>surnameLineEdit</tabstop>
  <tabstop>streetLineEdit</tabstop>
  <tabstop>cityLineEdit</tabstop>
  <tabstop>mailLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CustomerDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CustomerDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
